### PR TITLE
TDR-3660 remove feature block for ref generator

### PR DESCRIPTION
**Pull request by thanhz**

### CODE REVIEW

File: src/test/scala/uk/gov/nationalarchives/tdr/api/routes/FileRouteSpec.scala [hunk 1]
```diff
@@ -67,6 +70,7 @@ class FileRouteSpec extends TestContainerUtils with Matchers with TestRequest {
         1
       )
     }
+    val referenceMockServer = getReferencesMockSever(4)
```
Comment (TomJKing) [inline src/test/scala/uk/gov/nationalarchives/tdr/api/routes/FileRouteSpec.scala:73]: Should the number of references requested match the number of files, which is 5?

**Review by TomJKing — COMMENTED** [review]
Realise this is still draft, but had a question. Looks good. Thanks

File: src/test/scala/uk/gov/nationalarchives/tdr/api/routes/FileRouteSpec.scala (hunk 1 above)
Comment (thanhz) [reply src/test/scala/uk/gov/nationalarchives/tdr/api/routes/FileRouteSpec.scala:73]: The getReferencesMockServer method already creates a single reference. The parameter is how many additional references to create. I've renamed the method parameter to make it a bit clearer

File: src/test/scala/uk/gov/nationalarchives/tdr/api/routes/FileRouteSpec.scala [hunk 2]
```diff
@@ -206,4 +214,38 @@ class FileRouteSpec extends TestContainerUtils with Matchers with TestRequest {
     utils.createFile(UUID.fromString(fileFourId), consignmentId, fileName = "fileFourName", parentId = Some(UUID.fromString(folderId1)))
     utils.createFile(UUID.fromString(fileFiveId), consignmentId, fileName = "fileFiveName", parentId = Some(UUID.fromString(folderId1)))
   }
+
+  private def getReferencesMockSever(additionalRefs: Int = 0): WireMockServer = {
```
Comment (TomJKing) [inline src/test/scala/uk/gov/nationalarchives/tdr/api/routes/FileRouteSpec.scala:218]: Spelling, should be: `... MockServer`

**Review by TomJKing — COMMENTED** [review]
Sorry, just an additional very small comment. Thanks

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks.

**Pull request merged**